core-bundle/src/EventListener/DataContainer/PageRoutingListener.php

```diff
@@ -54,14 +54,13 @@ public function generateRouteConflicts(DataContainer $dc): string
             return '';
         }
 
-        $aliasPages = $pageAdapter->findSimilarByAlias($currentPage);
-
-        if (null === $aliasPages) {
+        if (!$aliasPages = $pageAdapter->findSimilarByAlias($currentPage)) {
             return '';
         }
 
         $conflicts = [];
-        $currentUrl = $this->buildUrl($currentPage->alias, $currentPage->urlPrefix, $currentPage->urlSuffix);
+        $currentRoute = $this->pageRegistry->getRoute($currentPage);
```

Comment:
what if the current page is not routable?

Reply:
Since the target of the callback is `fields.routeConflicts.input_field`, the listener will not be executed for non-routable pages because those do not have the `routeConflicts` field in their palette. Does this suffice or do you want me to add the following?

```php
if (!$this->pageRegistry->isRoutable($currentPage)) {
    return '';
}
```

Reply:
I would still add it, because we can't prevent people from adding that field to a palette on their page.

Reply:
Added in e092ade322ee0d47ef9747948d87f8b91ecb673f.